kafka-client/src/main/java/dev/responsive/kafka/config/ResponsiveDriverConfig.java

```diff
@@ -64,6 +64,12 @@ public class ResponsiveDriverConfig extends AbstractConfig {
       + "responsive server. This applies both to metadata requests and query execution.";
   private static final long REQUEST_TIMEOUT_MS_DEFAULT = 5000L;
 
+  // ------------------- other configurations ---------------------------------
+  public static final String PARTITION_EXPLODE_FACTOR_CONFIG = "responsive.partition"
```

Comment:
Can we keep configs on the same line and just put in a LineLength suppression for this class? This is one of the few cases it's important to keep everything together, imo

Reply:
+1 that's a great idea. this drove me nuts but for some reason I didn't think of the sensible thing...